src/pages/p/[handle].page.tsx

```diff
@@ -1,46 +1,166 @@
+import { paginateDepleteProjectsQueryCall } from 'utils/apollo'
+
+import Loading from '../../components/Loading'
+import V1Project from '../../components/v1/V1Project'
 
-import { layouts } from 'constants/styles/layouts'
 import { projectTypes } from 'constants/v1/projectTypes'
+import { layouts } from 'constants/styles/layouts'
 import { V1ArchivedProjectIds } from 'constants/v1/archivedProjects'
 
-import Loading from '../../components/Loading'
-import V1Project from '../../components/v1/V1Project'
+export const getStaticPaths: GetStaticPaths = async () => {
+  if (process.env.BUILD_CACHE_V1_PROJECTS === 'true') {
+    const projects = await paginateDepleteProjectsQueryCall({
+      variables: {
+        where: { cv_in: ['1', '1.1'] },
+      },
+    })
+    const paths = projects
+      .map(({ handle }) =>
+        handle
+          ? {
+              params: { handle },
+            }
+          : undefined,
+      )
+      .filter((i): i is { params: { handle: string } } => !!i)
+    return { paths, fallback: true }
+  }
+
+  return { paths: [{ params: { handle: 'juicebox' } }], fallback: true }
+}
 
-export default function V1Dashboard() {
+export const getStaticProps: GetStaticProps<{
+  metadata: ProjectMetadataV4
+  handle: string
+}> = async context => {
+  if (!context.params) throw new Error('params not supplied')
+  const handle = context.params.handle as string
+  const projects = await paginateDepleteProjectsQueryCall({
+    variables: {
+      where: { cv_in: ['1', '1.1'], handle },
+    },
+  })
+  if (!projects[0]?.metadataUri) {
+    throw new Error(`Failed to load metadata uri for ${context.params}`)
+  }
+  const url = ipfsCidUrl(projects[0].metadataUri)
+  try {
+    let metadata: ProjectMetadataV4 | undefined
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let error: any | undefined
+    do {
+      try {
+        if (error) {
+          // Back off for 3 secs - pinata rate limits for 180/min
+          const PINATA_RATE_LIMIT_SECONDS = 180
+          const SECONDS_IN_MINUTES = 60
+          await new Promise(resolve =>
+            setTimeout(
+              resolve,
+              (PINATA_RATE_LIMIT_SECONDS / SECONDS_IN_MINUTES) * 1000,
+            ),
+          )
+        }
+        const response = await axios.get(url)
+        metadata = consolidateMetadata(response.data)
+        error = undefined
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } catch (e: any) {
+        const status = e?.response?.status
+        if (status === 404 || status === 400) {
+          console.error('Page not found', {
+            url,
+            handle,
+          })
+          return { notFound: true }
+        }
+        console.warn('Failed to get url - backing off 3 seconds', {
+          status: status,
+          handle,
+          url,
+          error: e.message,
+        })
+        error = e
+      }
+    } while (!metadata)
```

Comment:
**suggestion** maybe we can move this into its own function `findProjectMetadata` or somethin, for readability.